### DISCUSSION TITLE
my Nvidia GPU was not detected

### DISCUSSION THREAD

**Discussion by huonide88** [intel/llvm#14595] [Q&A]
I have already installed oneAPI on Windows, but when I was using SYCL programming to check my available devices, my Nvidia GPU was not detected. So, do I need to install any additional plugins, and if so, how should I install them?

**Comment by DuncanMcBain** [comment]
Hi @huonide88 , if you are using the oneAPI kits from the Intel website, then you will also need to install the plugin from https://developer.codeplay.com/.
